### DISCUSSION TITLE
Visually mark reruns

### DISCUSSION THREAD

**Discussion by YakisobaRamen** [night/betterttv#6285] [Ideas]
A lot of streamers are doing 24h reruns. Since the last update it has become impossible to tell if someone is actually live or not, unless you hover the mouse cursor over their name and read their tag list.

I would be awsome if we had a feature that clearly visually marks reruns as such in the channel list. For example, the name of a channel running a rerun could be displayed in red, or with a box around it.

**Comment by night** [comment]
Reruns are indicated via a "Rerun" tag on the stream now — that's Twitch's native indicator for this. Closing.
